docs/tutorial_tips_tricks.rst

```diff
@@ -552,8 +552,10 @@ The following is an example of ``.gitlab-ci.yml`` file that does exactly that:
 It defines two stages.
 The first one, called ``generate``, will call ReFrame to generate the pipeline specification for the desired tests.
 All the usual `test selection options <manpage.html#test-filtering>`__ can be used to select specific tests.
-ReFrame will process them as usual, but instead of running the selected tests, it will generate the correct steps for running each test individually as a Gitlab job.
-We then pass the generated CI pipeline file to second phase as an artifact and we are done!
+ReFrame will process them as usual, but instead of running the selected tests, it will generate the correct steps
+for running each test individually as a Gitlab job. We then pass the generated CI pipeline file to second phase as
+an artifact and we are done! If ``image`` keyword is defined in ``.gitlab-ci.yml``, the emitted pipeline will use
+the same image as the one defined in the parent pipeline.
```

Comment:
We have a policy in the documentation to start new sentences in new lines for making diffs easier:

```suggestion
an artifact and we are done! 
If the ``image`` keyword is defined in ``.gitlab-ci.yml``, the emitted pipeline will use
the same image as the one defined in the parent pipeline.
The verbosity level of the initial invocation of ReFrame will also be retained in the invocations from the child pipeline.
```

Reply:
It'd be better if we had this doc enhancement in a separate PR.

Reply:
Ok. Will do that in the future PRs.